pkg/bootstrap/docker/up.go

```diff
@@ -714,14 +714,19 @@ func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
 
 // CheckAvailablePorts ensures that ports used by OpenShift are available on the Docker host
 func (c *CommonStartConfig) CheckAvailablePorts(out io.Writer) error {
+	userNamespace, err := c.DockerHelper().UserNamespaceEnabled()
```

Comment:
I think it's less robust if you rely on the caller/client to figure out if you have user namespaces turned on.